docs/features/exportpdf/README.md

```diff
@@ -0,0 +1,104 @@
+
+### Custom CSS rules
+
+Besides the inline styles, in order to mimic the editor content all the CKEditor 4 CSS rules are sent to the service. But let's say your company has a strict heading color policy: it has to be <span style="color:#4B0082">indigo</span> (`#4B0082`), definitely not <span style="color:#483D8B">dark slate blue</span> (`#483D8B`)! Fortunately you do not have to remember to change it every time. It is possible to attach your own custom CSS stylesheet via {@linkapi CKEDITOR.config.exportPdf_stylesheet CKEDITOR.config.exportPdf_stylesheet} configuration option or just add some styles with {@linkapi CKEDITOR.addCss} method.
+
+### Output file configuration
+
+A number of options like output file format or margins can be set in the {@linkapi CKEDITOR.config.exportPdf_options CKEDITOR.config.exportPdf_options} object. It is sent to the service along with the HTML and CSS and processed on the server side. To check out the possible configurations, visit the [service documentation](https://pdf-converter.cke-cs.com/docs).
+
+### Data preprocessing
+
+Plugin provides a custom {@linkapi CKEDITOR.editor#exportPdf exportPdf event}. It can be used for custom data processing (e.g. to ensure the output text will be black, not <span style="color:pink;background-color:yellow">pink on yellow background</span>). Editor uses it too, so just remember to add the right priority to the listener:
+
+* 1-14: Data is available in the original string format.
+* 15: Data is processed by the plugin.
+* 16-19: Data that will be sent to the service can be modified.
+* 20: Data is sent to the service.
+
+### Asynchronous preprocessing
+
+It is even possible to run some asynchronous tasks before sending data to the server. To do this simply stop the event and set the flag telling that process is not finished yet. After it is done, remove flag and refire the `exportPdf` event:
+
+	editor.on( 'exportPdf', function( evt ) {
+		// Stop the event if the async process is still on.
+		if ( !evt.data.asyncDone ) {
+			evt.cancel();
+		} else {
+			delete evt.data.asyncDone;
+		}
+	} );
+
+	editor.on( 'exportPdf', function( evt ) {
+		// Here e.g. some AJAX request can be sent; after it is done set the flag to `true`:
+		evt.data.asyncDone = true;
+
+		// Refire the event.
+		editor.fire( 'exportPdf', evt.data );
+	}, null, null, 1 );
+
+### Relative vs absolute URLs
+
+Assets like images can be attached using relative URLs, but before data is sent to the service, relative links are converted to absolute ones. In some cases it will mean that data will not be accessible by the server (e.g. if it is referenced locally or through the infranet) - remember to expose such assets globally.
```

Comment:
```suggestion
Assets like images can be attached using relative URLs, but before data is sent to the service, relative links are converted to absolute ones. In some cases it will mean that data will not be accessible by the server (e.g. if it is referenced locally or through the infranet) - remember to expose such assets publicly or use absolut URLs to publicly available assets.
```

Maybe we could also mention [`baseHref`](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-baseHref). So one can have assets storage exposed somewhere else?